app/assets/javascripts/cqm/ValueSet.js

```diff
@@ -7,18 +7,15 @@ const [String, Mixed, ObjectId] = [
   mongoose.Schema.Types.ObjectId,
 ];
 
-const ValueSetSchema = mongoose.Schema(
+const ValueSetSchema = new mongoose.Schema(
   {
     oid: String,
     display_name: String,
     version: String,
-    categories: Mixed,
 
     concepts: [Concept.ConceptSchema],
 
-    user: { type: ObjectId, ref: 'User', index: true }, // Bonnie-specific
-    bundle: { type: ObjectId, ref: 'Bundle' }, // Cypress-specific
-    measures: [{ type: ObjectId, ref: 'Measure' }],
+    measures: [{ type: ObjectId, ref: 'Measure', index: true }],
```

Comment:
Is this needed given the `has_and_belongs_to_many :value_sets, inverse_of: nil` in the measure model?

Reply:
Removed it.